Empire/Empire.cpp

```diff
@@ -107,6 +112,38 @@ namespace {
         return retval;
     }
 
+    float CalculateNewStockpile(int empire_id, int stockpile_location_id, float starting_stockpile,
+                                         const std::map<std::set<int>, float>& available_pp,
+                                         const std::map<std::set<int>, float>& allocated_pp,
+                                         const std::map<std::set<int>, float>& allocated_stockpile_pp)
+    {
+        const Empire* empire = GetEmpire(empire_id);
+        if (!empire) {
+            ErrorLogger() << "CalculateStockpileContribution() passed null empire.  doing nothing.";
+            return 0.0f;
+        }
+        float stockpile_transfer_efficiency = empire->GetMeter("METER_IMPERIAL_PP_TRANSFER_EFFICIENCY")->Current();
+        // locally, within same resource group as the stockpile location (empire capital) we currently allow stockpile contribution at no penalty
+        float local_stockpile_transfer_rate = 1.0;
```

Comment:
As discussed, remove, or at least disable, the concept of a stockpile location

Reply:
I didnt get as much community feedback as i hoped so i simplified the stockpile code in my branch not to use the stockpile location. Stockpile still has dead code but i will do a cleanup PR after people actually used the stockpile.